build_latest.sh

```diff
@@ -24,18 +24,20 @@ source ./common_functions.sh
 # shellcheck source=dockerfile_functions.sh
 source ./dockerfile_functions.sh
 
-if [ $# -ne 3 ]; then
+if [ $# -ne 4 ]; then
 	echo
-	echo "usage: $0 version vm package"
+	echo "usage: $0 version vm package runtype"
 	echo "version = ${supported_versions}"
 	echo "vm      = ${all_jvms}"
 	echo "package = ${all_packages}"
+    echo "runtype = ${all_runtypes}"
```

Comment:
Looks like spaces instead of tabs

Reply:
Changed to tabs, Thanks for pointing it out.